api/server/router/network/network_routes.go

```diff
@@ -100,6 +100,20 @@ func (e ambigousResultsError) Error() string {
 
 func (ambigousResultsError) InvalidParameter() {}
 
+type conflictingNetworkName struct {
```

Comment:
see [#34484(comment)](https://github.com/moby/moby/pull/34484#issuecomment-325014066)

Reply:
Yes, I have use it, and this is the implementation of the `errdefs` interfaces.